app/models/pod.js

```diff
@@ -99,7 +99,13 @@ var Pod = Resource.extend(DisplayImage, {
       return resource;
     }
   }),
-
+  image: function () {
```

Comment:
This is what the DisplayImage mixin that's loaded does as `displayImage` (except it says that there's multiple instead of picking one)